source/api/presentation/3.0/index.md

```diff
@@ -1088,11 +1088,11 @@ The Canvas represents an individual page or view and acts as a central point for
 
-A Canvas _MUST_ have a rectangular aspect ratio (described with the `height` and `width` properties) and/or a `duration` to provide an extent in time. These dimensions allow resources to be associated with specific regions of the Canvas, within the space and/or time extents provided. Content _MUST NOT_ be associated with space or time outside of the Canvas's dimensions, such as at coordinates below 0,0, greater than the height or width, before 0 seconds, or after the duration.
+Content resources are associated with the Canvas via Web Annotations.  Content that is to be rendered as part of the Canvas _MUST_ be associated by an Annotation with the "painting" `motivation`. These Annotations are recorded in the `items` of one or more AnnotationPages, refered to in the `items` array of the Canvas. If, according to the organization providing the Manifest, clients _SHOULD_ render the Annotation quickly then it _SHOULD_ be embedded within the Manifest directly.  Other AnnotationPages can be referenced with just their `id`, `type` and optionally a `label`, and clients _SHOULD_ dereference these pages to discover further content.  Content in this case includes media assets such as images, video and audio, textual transcriptions or editions of the Canvas, as well as commentary about the object represented by the Canvas.  These different uses _MAY_ be split up across different AnnotationPages.
 
-Renderers _MUST_ scale content into the space represented by the Canvas, and _SHOULD_ follow any `timeMode` adjustment provided for time-based media.  If the Canvas represents a view of a physical object, the spatial dimensions of the Canvas _SHOULD_ be the same scale as that physical object, and images _SHOULD_ depict only the object.
+A Canvas _MUST_ have a rectangular aspect ratio (described with the `height` and `width` properties) and/or a `duration` to provide an extent in time. These dimensions allow resources to be associated with specific regions of the Canvas, within the space and/or time extents provided. Content _MUST NOT_ be associated with space or time outside of the Canvas's dimensions, such as at coordinates below 0,0, greater than the height or width, before 0 seconds, or after the duration. Content resources that have dimensions which are not defined for the Canvas _MUST NOT_ be associated with that Canvas. For example, it is an error to associate a Video resource (which has height, width and duration) with a Canvas that does not have all three dimensions, but it is _not_ an error to use a "painting" Annotation to associate an Image (which has only height and width) with a Canvas that has all three dimensions. Such a resource _SHOULD_ instead be referenced with the `rendering` property, or by Annotations with a `motivation` other than "painting", in the `annotations` property.
```

Comment:
I don't know why this didn't stick out before, but the last sentence here ("Such a resource _SHOULD_...") refers to the resource mentioned in the first clause of the preceding sentence (the error case) rather than the resource mentioned in the second, which now looks confusing to me. We could swap the resources round so that the last sentence refers to the resource just described:

> For example, it is valid to use a "painting" Annotation to associate an Image (which has only height and width) with a Canvas that has height, width and duration, but it is an error to associate a Video resource (which has height, width and duration) with a Canvas that only has height and width. Such a resource _SHOULD_ instead be referenced with the `rendering` property, or by Annotations with a `motivation` other than "painting", in the `annotations` property.

Reply:
👍